Sources/StreamChatUI/ChatThread/ChatThreadVC.swift

```diff
@@ -33,6 +33,9 @@ open class ChatThreadVC: _ViewController,
         channelController.client
     }
 
+    /// The throttler to make sure that the marking read is not spammed.
+    var markReadThrottler: Throttler = Throttler(interval: 1, queue: .main)
```

Comment:
shouldn't this be 3 seconds?

Reply:
Usually threads are not that busy in comparison with regular channels, so IMO 3 seconds is unnecessary (IMO even for channels is unnecessary, but that is another discussion that already had 😄 )